### DISCUSSION TITLE
Merge my attributes when inserting jobs in the table

### DISCUSSION THREAD

**Discussion by foremtehan** [laravel/framework#47636] [Ideas]
I want to add custom columns to `jobs` table, but right now there is no ways that make me able to merge my attributes.

Suggestion:

```php
Queue::mergeAttibutesWhenDispatchingJobs(function () {
    return [
        'custom_column' => 'my value',
    ];
});
```

**Comment by henzeb** [comment]
What is the use case?

**Reply by foremtehan** [reply]
I have a discord bot that dispatch a job on every user requests, I want to tell user you in the queue and your position is X, There isn't an easy to do this you have read payload and so on ...
